applications/FluidDynamicsApplication/custom_utilities/fluid_calculation_utilities.cpp

```diff
@@ -0,0 +1,75 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+// External includes
+
+// Project includes
+
+// Include base h
+#include "custom_utilities/fluid_calculation_utilities.h"
+
+namespace Kratos
+{
+
+template<>
+void FluidCalculationUtilities::AssignValue(
+    array_1d<double, 2>& rOutput,
+    const array_1d<double, 3>& rInput)
+{
+    rOutput[0] = rInput[0];
+    rOutput[1] = rInput[1];
```

Comment:
```suggestion
    rOutput[1] = rInput[1];
    rOutput[2] = 0.0;
```
I think this is the expected behavior. Aside of this is safer in case the array is not initialized.